bin/utils/cli-config-utils.js

```diff
@@ -6,6 +6,28 @@ const tildify = require('tildify');
 const color = require('colorette');
 const argv = require('getopts')(process.argv.slice(2));
 
+function parseConfigObj(opts) {
+  const config = { migrations: {} };
+
+  if (opts.client) {
+    config.client = opts.client;
+  }
+
+  if (opts.connection) {
+    config.connection = opts.connection;
+  }
+
+  if (opts.migrationsDirectory) {
```

Comment:
I wonder if there is a way to make all overrides case-insensitive. Probably we can do check on lowercased entry pairs.

Reply:
As I can understand, the goal is to lower case the properties in options object before merge with config in knexfile ?

Reply:
But some options are camelCase, no?

Reply:
Mmh, I will check that this week.